dev/bin/minikube-start.sh

```diff
@@ -42,17 +42,15 @@ minikube -p "${MINIKUBE_PROFILE}" addons enable ingress-dns
 sleep 5
 
 if [ "${disable_extra_components}" != 'true' ]; then
-  kustomize build ${KUSTOMIZE_DIR}/overlays/minikube/keycloak | kubectl apply -f -
+  kustomize build ${KUSTOMIZE_DIR}/overlays/local/keycloak | kubectl apply -f -
```

Comment:
Yes, it is just for CI. I can rename `minikube` to `ci` overlay

Reply:
@sutaakar Those name changes make complete sense to me 👍  Could you please send an email to the team on `openbridge-dev` when this PR is ready to merge so that everyone is aware of the changes.

cheers,

Rob